data/ecosystem/vendors.yaml

```diff
@@ -294,6 +294,12 @@
   contact:
   oss: false
   commercial: true
+- name: OllyGarden
+  nativeOTLP: true
+  url: https://ollygarden.com
+  contact:
```

Comment:
can you put a point of contact here, can just be your github handle (preferred) so if we ever change something here we can tag you.